etc/kayobe/ansible/growroot.yml

```diff
@@ -27,57 +27,60 @@
     growroot_vg: "rootvg"
     # Don't assume facts are present.
     os_family: "{{ ansible_facts.os_family | default('Debian' if os_distribution == 'ubuntu' else 'RedHat') }}"
+    # LVM override
+    growroot_skip_lvm_check: false
 
   tasks:
-    - name: Check if growpart is installed
+    - name: Check LVM status
       shell:
-        cmd: type growpart
+        cmd: vgdisplay | grep -q lvm2
```

Comment:
FWIW, there may be cases where this is not installed, but I think this will do the right thing anyway.